modules/spring-config/src/test/java/org/mule/runtime/config/spring/SpringXmlConfigurationMuleArtifactFactoryTest.java

```diff
@@ -19,10 +31,8 @@
 import org.dom4j.Namespace;
 import org.dom4j.QName;
 import org.dom4j.io.DOMWriter;
-import org.hamcrest.CoreMatchers;
-import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
 import org.w3c.dom.Element;
 
 public class SpringXmlConfigurationMuleArtifactFactoryTest {
```

Comment:
Not because of your changes, but can you rename it to SpringXmlConfigurationMuleArtifactFactoryTestCase and extend from AbstractMuleTestCase?